README.rst

```diff
@@ -0,0 +1,158 @@
+https://docs.docker.com/install/linux/docker-ce/debian/>`_, `nvidia-docker <https://github.com/NVIDIA/nvidia-docker>`_, and `CUDA <https://developer.nvidia.com/cuda-downloads>`_ separately.
+
+Build a local docker container, specifying the tensorflow version with TF_VERSION
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    git clone https://github.com/vanvalenlab/deepcell-tf.git
+    cd deepcell-tf
+    docker build --build-arg TF_VERSION=1.15.0-gpu -t $USER/deepcell-tf .
+
+
+Run the new docker image
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    # NV_GPU refers to the specific GPU to run DeepCell on, and is not required
+
+    NV_GPU='0' nvidia-docker run -it \
+    -p 8888:8888 \
+    $USER/deepcell-tf:latest
+
+It can also be helpful to mount the local copy of the repository and the scripts to speed up local development. However, if you are going to mount a local version of the repository, you must first run the docker image without the local repository mounted so that the c extensions can be compiled and then copied over to your local version.
+
+.. code-block:: bash
+
+    # First run the docker image without mounting externally
+    NV_GPU='0' nvidia-docker run -it \
+    -p 8888:8888 \
+    $USER/deepcell-tf:latest
+
+    # Use ctrl-p, ctrl-q to exit the running docker image without shutting it down
+
+    # Then, get the container_id corresponding to the running image of deepcell
+    container_id=$(docker ps -q --filter ancestor="$USER/deepcell-tf")
+
+    # Copy the compiled c extensions into your local version of the codebase:
+    docker cp "$container_id:/usr/local/lib/python3.6/dist-packages/deepcell/utils/compute_overlap.cpython-36m-x86_64-linux-gnu.so" deepcell/utils/compute_overlap.cpython-36m-x86_64-linux-gnu.so
+
+    # close the running docker
+    docker kill $container_id
+
+    # you can now start the docker image with the code mounted for easy editing
+    NV_GPU='0' nvidia-docker run -it \
+    -p 8888:8888 \
+    -v $PWD/deepcell:/usr/local/lib/python3.6/dist-packages/deepcell/ \
+    -v $PWD/scripts:/notebooks \
+    -v /$PWD:/data \
+    $USER/deepcell-tf:latest
+
+How to Cite
+-----------
+* `The original DeepCell paper <https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005177>`_
+* `DeepCell 2.0: Automated cloud deployment of deep learning models for large-scale cellular image analysis <https://www.biorxiv.org/content/early/2018/12/22/505032.article-metrics>`_
+
+Copyright
+---------
+
+Copyright © 2018-2020 `The Van Valen Lab <http://www.vanvalen.caltech.edu/>`_ at the California Institute of Technology (Caltech), with support from the Paul Allen Family Foundation, Google, & National Institutes of Health (NIH) under Grant U24CA224309-01.
```

Comment:
no current year fancy trick for this one? Not needed but would be cool.

Reply:
I'll keep an eye out, but the previous one only worked because it was in a .py file